viewer/src/main/webapp/viewer-html/common/viewercontroller/ViewerController.js

```diff
@@ -145,6 +145,9 @@ Ext.define("viewer.viewercontroller.ViewerController", {
             this.mapComponent = new viewer.viewercontroller.FlamingoMapComponent(this, mapId,config);
         }else if(viewerType === "openlayers") {
             this.mapComponent = new viewer.viewercontroller.OpenLayersMapComponent(this, mapId,config);
+        }else if(viewerType === "ol"){
```

Comment:
Do not really like the "ol" viewerType. This will be very difficult to find / replace. Why not "openlayers5" for example? Easier to find, easier to replace